arangod/Aql/Functions.cpp

```diff
@@ -5587,44 +5587,47 @@ AqlValue Functions::GeoDistance(ExpressionContext* exprCtx,
 
 /// @brief function GEO_IN_RANGE
 AqlValue Functions::GeoInRange(ExpressionContext* ctx,
-                               AstNode const&,
+                               AstNode const& node,
                                VPackFunctionParameters const& args) {
   TRI_ASSERT(ctx);
-  constexpr char const AFN[] = "GEO_IN_RANGE";
+  TRI_ASSERT(aql::NODE_TYPE_FCALL == node.type);
+
+  auto const* impl = reinterpret_cast<arangodb::aql::Function*>(node.getData());
```

Comment:
This can be changed to a simple static_cast.
```suggestion
  auto const* impl = static_cast<arangodb::aql::Function*>(node.getData());
```